core/src/main/scripts/importer/validateData.py

```diff
@@ -2329,6 +2330,52 @@ def checkLine(self, data):
         else:
             self.fusion_entries[fusion_entry] = self.line_number
 
+
+class StructuralVariationValidator(Validator):
+
+    """Basic validation for fusion data. Validates:
+
+    1. Required column headers
+    2. Gene IDs
+    3. If there is a second gene, also validate Gene IDs of second gene
+    """
```

Comment:
please also validate
- if either a position is given OR an exon.
- if a transcript id is given, if it is a valid ENSEMBL id (will we get transcript ids in our data?)

Reply:
Current design: fusion events require exon columns and values. 

I implemented validation for exon to be in the transcript. 

Todo: validate transcript.

Reply:
Todo: validate transcript.

Reply:
Done. I implemented validation for transcripts.